aiden/class1-1/2438.py

```diff
@@ -0,0 +1,8 @@
+#2438
+
+n = int(input())
+
+for i in range(n):
+    for j in range(i+1):
+        print('*',end="")
+    print()
```

Comment:
```suggestion
for i in range(1, N+1):
    print('*' * i)
```

제 생각에는 이렇게 하는게 더 간단하더라구요.
aiden님의 방법도 좋네요!